src/Neo.VM/OpCode.cs

```diff
@@ -878,5 +878,14 @@ public enum OpCode : byte
         CONVERT = 0xDB,
 
         #endregion
+
+        /// <summary>
+        /// Turns the vm state to FAULT immediately, and cannot be caught. Includes a reason
+        /// </summary>
+        ABORTMSG = 0xE0,
```

Comment:
If we will make versioning, because previously these opcodes fault, and now don't, why don't add these arguments to the previous opcodes if the version is the last one?